test/with_dummyserver/test_poolmanager.py

```diff
@@ -469,8 +469,20 @@ def test_top_level_request_with_timeout(self):
                 redirect=True,
                 retries=None,
                 timeout=2.5,
+                chunked=False,
+                body_pos=None,
+                assert_same_host=True,
             )
 
+    def test_top_level_request_with_chunked(self):
+        res = request(
+            "GET", f"{self.base_url}/chunked", chunked=True, preload_content=False
```

Comment:
`chunked` refers to how the request body is encoded, not the response body. You need to specify `body=<something>` for it to do something.